src/address.rs

```diff
@@ -145,11 +158,6 @@ impl<A, Rc: RefCounter> Address<A, Rc> {
         self.len() == 0
     }
 
-    /// Convert this address into a generic address which can be weak or strong.
-    pub fn as_either(&self) -> Address<A, Either> {
```

Comment:
I'm not sure it's an advantage to take this off of Address<A, Either>, as it means that it might be harder to write code generic over all refcounters

Reply:
The problem is that if we want to have it here, we need a conversion function on `RefCounter` again so I can call it from any ref counter.

Doable but I figured it is unnecessary. There are only two types of ref counter this is relevant for so it is effectively the same public API.

Reply:
This is of course different if we want users to be able to call this from an extension trait.

Reply:
I think regardless there should be a way, in the case of generic impls over RefCounter

Reply:
I've reintroduced `RefCounter::into_either` in [`40fd855` (#158)](https://github.com/Restioson/xtra/pull/158/commits/40fd85596eafd4dc0b11dda55eac0d29ba60258a).

I've also opened https://github.com/Restioson/xtra/pull/171 which adds a test that these APIs should exist.